src/pybind/mgr/cephadm/services/nfs.py

```diff
@@ -111,10 +114,9 @@ def get_set_cmd_dicts(out: str) -> List[dict]:
 
 class NFSGanesha(object):
     def __init__(self,
-                 mgr,
-                 daemon_id,
-                 spec):
-        # type: (cephadm.CephadmOrchestrator, str, NFSServiceSpec) -> None
+                 mgr: "CephadmOrchestrator",
+                 daemon_id: str,
+                 spec: NFSServiceSpec) -> None:
```

Comment:
fyi, there is a tool called `com2ann` which worked pretty well for me.

Reply:
> fyi, there is a tool called `com2ann` which worked pretty well for me.

neat! that does make these much easier to resolve.